Examples/ObjectiveC/InsetMapExample.m

```diff
@@ -0,0 +1,71 @@
+#import "InsetMapExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleInsetMap = @"InsetMapExample";
+
+@interface InsetMapExample () <MGLMapViewDelegate>
+@property (nonatomic) MGLMapView *mapView;
+@property (nonatomic) MGLMapView *miniMapView;
+@end
+
+@implementation InsetMapExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    // Set the map's center coordinate
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(59.31, 18.06)
+                zoomLevel:9
+                 animated:NO];
+    
+    self.miniMapView = [[MGLMapView alloc] initWithFrame:CGRectZero];
+    [self.miniMapView setAllowsScrolling:NO];
+    [self.miniMapView setAllowsTilting:NO];
+    [self.miniMapView setAllowsZooming:NO];
+    [self.miniMapView setAllowsRotating:NO];
+
+    [self.miniMapView.logoView setHidden:YES];
+    self.miniMapView.attributionButton.tintColor = UIColor.clearColor;
+     UIColor *black = [UIColor blackColor];
+    [self.miniMapView.layer setBorderColor: black.CGColor];
+    [self.miniMapView.layer setBorderWidth:1];
+    
+    [self.miniMapView setCenterCoordinate:self.mapView.centerCoordinate zoomLevel:self.mapView.zoomLevel animated:NO];
+    [self.miniMapView setTranslatesAutoresizingMaskIntoConstraints:NO];
+
+    // Set the delegate property of our map view to self after instantiating it
+    self.mapView.delegate = self;
+
+    [self.view addSubview:self.mapView];
+    [self.view addSubview:_miniMapView];
+    [self installConstraints];
+
+    
+}
+
+- (void)mapView:(MGLMapView *)mapView regionWillChangeAnimated:(BOOL)animated {
+    [self.miniMapView setCamera:self.mapView.camera animated:NO
+     ];}
```

Comment:
Nit: Extra spacing before `];}` - also the `;` doesn't need to be there. The end of the array should be placed on line 49 as well.

Reply:
xCodee won't let me remove the `;` :(

Reply:
Ah, my bad - I just noticed the `;` is part of the ending statement of `setCamera`. You most recent change corrected this.